examples/nurbs.sid

```diff
@@ -0,0 +1,67 @@
+ # Evaluates an point on a nurbs curve
+
+Point { x: 0.0, y: 0.0 } def!
```

Comment:
This type specification isn't quite complete. I think you meant for it to be `{ x: float, y: float }`.

More notably, I'm thinking of using `const` instead of `def` for global definitions. (so that def can be used for scope local definitions, as you use it within sub-stacks/functions)

Reply:
Const since it wouldn't be allowed to change it. Or maybe global, since other definitions won't be accessible even in sub-scopes?.... Naming is hard...